script.js

```diff
@@ -0,0 +1,140 @@
+// Mobile Menu
+
+const hamburger = document.querySelector('#hamburger');
+const closeMenu = document.querySelector('#glyphicon-remove');
+const mobileMenu = document.querySelector('#mobile-menu');
+const menuLinks = document.querySelectorAll('#mobile-menu li');
+
+function toggleMenu() {
+  mobileMenu.classList.toggle('hidden');
+}
+
+hamburger.addEventListener('click', toggleMenu);
+closeMenu.addEventListener('click', toggleMenu);
+
+menuLinks.forEach((link) => {
+  link.addEventListener('click', toggleMenu);
+});
```

Comment:
- Hey, please, let's check our mobile menu as it doesn't seem to work when clicked. let's fix this issue. :pray: 
- Add the missing script in the about.html file `<script src="script.js"></script>`
- Let's also kindly increase the size of the hamburger to improve the user experience as the size is too small compared to the design template.